src/main/kotlin/com/sourcegraph/cody/chat/AgentChatSession.kt

```diff
@@ -299,7 +305,6 @@ private constructor(
             when (commandId) {
               CommandId.Explain -> agent.server.legacyCommandsExplain()
               CommandId.Smell -> agent.server.legacyCommandsSmell()
-              CommandId.Test -> agent.server.legacyCommandsTest()
```

Comment:
pls update TESTING.md btw (at least remove the old command test desc):
https://sourcegraph.com/-/editor?remote_url=github.com%2Fsourcegraph%2Fjetbrains&branch=main&file=TESTING.md&start_row=182&start_col=0&end_row=202&end_col=7&editor=JetBrains&version=v5.5.9